src/camera/projection_math.jl

```diff
@@ -312,43 +312,44 @@ end
 
 # project between different coordinate systems/spaces
 
-function space_to_clip(cam::Camera, space::Symbol)
+function space_to_clip(cam::Camera, space::Symbol, projectionview=true)
+    return to_value(space_to_clip_obs(cam::Camera, space::Symbol, projectionview))
+end
+
+function space_to_clip_obs(cam::Camera, space::Symbol, projectionview::Bool)
     if is_data_space(space)
-        cam.projectionview[]
+        return projectionview ? cam.projectionview : cam.projection
     elseif is_pixel_space(space)
-        cam.pixel_space[]
+        return cam.pixel_space
     elseif is_relative_space(space)
-        Mat4f(2, 0, 0, 0, 0, 2, 0, 0, 0, 0, 1, 0, -1, -1, 0, 1)
+        return Mat4f(2, 0, 0, 0, 0, 2, 0, 0, 0, 0, 1, 0, -1, -1, 0, 1)
     elseif is_clip_space(space)
-        Mat4f(I)
+        return Mat4f(I)
     else
         error("Space $space not recognized. Must be one of $(spaces())")
     end
 end
```

Comment:
Relative and clip space should also return observables

Reply:
Actually, it seems the observable version isn't needed... I'll remove it